.NET/Microsoft.Recognizers.Text.sln

```diff
@@ -812,6 +812,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Turkish", "Turkish", "{2F41
 		..\Specs\DateTime\Turkish\TimePeriodParser.json = ..\Specs\DateTime\Turkish\TimePeriodParser.json
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tools", "Tools", "{F6B54840-FCA6-4824-A79B-B05B186B0DA8}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "FormatCheckTool", "..\Tools\src\FormatCheckTool\FormatCheckTool.csproj", "{D8836462-EBA2-4DF2-B23D-17996776FF96}"
```

Comment:
I'm fine with adding the tool in the overall .NET solution, especially for CI. But I suggest also creating a standalone solution only with the tool in its directory. People changing the tool may not need to load everything.